src/rust/engine/fs/src/store.rs

```diff
@@ -982,6 +982,15 @@ c0033144c785a94d3ebd82baa931cd16";
       .wait()
   }
 
+  pub fn load_file_bytes2(
```

Comment:
not 100% sure on the convention, but I think that numeric suffixes like this are used for deprecations, rather than overloads (ie `do_thing2` deprecates `do_thing`). But I don't know what a great name would be here... `load_file_bytes_store`...? ugh.

An (overly ambitious, probably) alternative would be to split the traits further and implement a smaller `load_file_bytes` trait for both `Store` and `ByteStore`.

Reply:
I miss overloads :(

Renamed to load_file_bytes_from_store for now :)